/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:38769'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:38752'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:34820'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:44344'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:38069'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:40517'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:34513'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:36119'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:39129'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:34650'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:45455'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:36598'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:46657'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:37864'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:41605'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:34219'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:41045'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:43923'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:34202'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:38997'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:36788'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:36255'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:34421'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:37026'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:41746'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:38786'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:43380'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.5.142:34179'
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:39793
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:39793
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:40208
distributed.worker - INFO -          dashboard at:         172.30.5.142:45981
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:40244
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:40244
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:40208
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:44529
distributed.worker - INFO -          dashboard at:         172.30.5.142:43967
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:35223
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:35223
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          dashboard at:         172.30.5.142:40469
distributed.worker - INFO -                Memory:                    4.29 GB
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:43089
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:44529
distributed.worker - INFO -          dashboard at:         172.30.5.142:41039
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-ff8xcu7a
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:43089
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          dashboard at:         172.30.5.142:40354
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:41232
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:34344
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:45154
distributed.worker - INFO -          dashboard at:         172.30.5.142:42917
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-iw2eyf_h
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:37668
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:34344
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:45154
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:41232
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:37668
distributed.worker - INFO -                Memory:                    4.29 GB
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:42307
distributed.worker - INFO -          dashboard at:         172.30.5.142:36508
distributed.worker - INFO -          dashboard at:         172.30.5.142:45634
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-40y_b2ky
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          dashboard at:         172.30.5.142:34410
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          dashboard at:         172.30.5.142:38121
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:42307
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:36392
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-ut07a_pw
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-q8awafdf
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:36392
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO -          dashboard at:         172.30.5.142:36713
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          dashboard at:         172.30.5.142:40098
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-rshnaoyn
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-cgjpagrh
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-qs6cp9jx
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-ner4n5zw
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:34408
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-m8kv91h0
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-vwzek9ng
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:34408
distributed.worker - INFO -          dashboard at:         172.30.5.142:41591
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-ss1ri2_9
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-tf_cw7b6
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:43999
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:43999
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          dashboard at:         172.30.5.142:37590
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-wwkn0fn8
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:33225
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:33225
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:37962
distributed.worker - INFO -          dashboard at:         172.30.5.142:38464
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:37962
distributed.worker - INFO -          dashboard at:         172.30.5.142:43743
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-v4styxxj
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-nye93iai
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:45304
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:45304
distributed.worker - INFO -          dashboard at:         172.30.5.142:33802
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:40839
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:38700
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:38700
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:40839
distributed.worker - INFO -          dashboard at:         172.30.5.142:34620
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          dashboard at:         172.30.5.142:34649
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-8vfi8tmz
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-g04q87mm
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-p42nxs66
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:36821
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:37176
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:36821
distributed.worker - INFO -          dashboard at:         172.30.5.142:41806
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:37176
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          dashboard at:         172.30.5.142:42263
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-87dpkgux
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:46438
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:46438
distributed.worker - INFO -          dashboard at:         172.30.5.142:44723
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-mv2v2rho
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-qc22lcu7
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:37078
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:37078
distributed.worker - INFO -          dashboard at:         172.30.5.142:33548
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-xfd1frrd
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:43895
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:43895
distributed.worker - INFO -          dashboard at:         172.30.5.142:43433
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-6mlojfif
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:38852
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:38852
distributed.worker - INFO -          dashboard at:         172.30.5.142:40936
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-tt3ksw9p
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:40508
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:40508
distributed.worker - INFO -          dashboard at:         172.30.5.142:36965
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:39151
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:39151
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          dashboard at:         172.30.5.142:34450
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-i6m3_quc
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-bri7pq8t
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:   tcp://172.30.5.142:43706
distributed.worker - INFO -          Listening to:   tcp://172.30.5.142:43706
distributed.worker - INFO -          dashboard at:         172.30.5.142:38157
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-ewe59jqa
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO -         Registered to:   tcp://172.30.100.3:41696
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.utils_perf - INFO - full garbage collection released 31.88 MB from 233 reference cycles (threshold: 10.00 MB)
distributed.core - INFO - Event loop was unresponsive in Worker for 3.28s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:34344
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:33225
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:35223
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:34408
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:36392
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:36821
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:37962
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:37078
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:38700
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:37668
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:37176
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:38852
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:40208
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:39793
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:41232
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:39151
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:42307
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:40839
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:40244
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:40508
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:43895
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:43706
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:43089
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:45304
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:44529
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:46438
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:45154
distributed.worker - INFO - Stopping worker at tcp://172.30.5.142:43999
slurmstepd: error: *** JOB 7324643 ON n2117 CANCELLED AT 2019-09-03T16:40:24 DUE TO TIME LIMIT ***
